command_line_tool/command_line.cc

```diff
@@ -113,6 +115,75 @@ namespace commandline {
     return profile_result;
   }
 
+// Parse hashtag string 
+  std::vector<std::string> CommandLineClient::ParseHashtag(std::string raw_hashtag) {
+    LOG(INFO) << "In command_line_client, parsing hashtag";
+    std::vector<std::string> words; 
+
+    std::stringstream ss1(raw_hashtag); 
+    std::string hashtag; 
+      
+    while(std::getline(ss1, hashtag, ' ')) { 
+      if(hashtag[0] == '#' && hashtag.size() > 1){
```

Comment:
What will this method return for raw hashtag "#hello #world"? The backend should be able to support multi-hashtag search if there are more than one hashtag.